### DISCUSSION TITLE
in the post installation for LS210, i install mdadm, and get command not found

### DISCUSSION THREAD

**Discussion by vujnovich** [1000001101000/Debian_on_Buffalo#208] [Q&A]
good day! I have the ls210 and the xfs was setup from the stock firmware... so i booted up and the post installation says

   ~# mdadm --detail --scan
   ARRAY /dev/md0 metadata=0.90 UUID=8de323cf:21f920b7:2776d444:be501e4f  
   ARRAY /dev/md1 metadata=1.2 UUID=e82d03b7:a7afdb01:663e8e97:125fc7c9  
   ARRAY /dev/md20 metadata=1.2 UUID=116997d2:e7f48544:24542b4e:8e4c2e18

i try and run that and get" bash: mdadm: command not found"

i did install sudo like in the novaspirit tech. but not anything else.  No idea how to access my files or mount it. im stuck on the very beginning right after i install mdadm

**Comment by 1000001101000** [comment]
that mean mdadm isn't on the current user's path. Try one of the following first:
`su - root`
or
`sudo su -`
